scout/server/blueprints/cases/views.py

```diff
@@ -313,6 +313,19 @@ def pdf_case_report(institute_id, case_name):
     )
 
 
+@cases_bp.route("/<institute_id>/<case_name>/chanjo2_coverage_report", methods=["GET"])
+async def chanjo2_coverage_report(institute_id, case_name):
+    """Return the HTML coverage report created by chanjo2."""
+
+    institute_obj, case_obj = institute_and_case(store, institute_id, case_name)
+    report_html_content: str = await controllers.chanjo2_coverage_report_contents(
```

Comment:
Does it really make a difference here, since you return the whole content as html?

Reply:
You have to have await because it's calling an async function. Or perhaps I didn't understand the question?

Reply:
Well, I could have put the comment on the function instead. As far as I can tell it's one query, one response, one page - nothing really being concurrent? Am I missing something and there is real application here, or just other default timeouts?

Reply:
Ah! The chanjo2 API call might have long execution time, according to the numbers of genes involved. I's not in place yet but I think might add a background execution of the report creation.

Reply:
Sure, let's do like that. I can re-test the whole thing on our stage server